packages/react/src/_popover/Popover.tsx

```diff
@@ -97,33 +119,59 @@ export function usePopover<RT extends ReferenceType = ReferenceType>(
 		],
 		// Ensures the floating element remains anchored to its reference element
 		// https://floating-ui.com/docs/react#anchoring
-		whileElementsMounted(referenceEl, floatingEl, update) {
-			const cleanup = autoUpdate(referenceEl, floatingEl, update, {
+		...(!hiddenWithCSS && {
+			whileElementsMounted(referenceEl, floatingEl, update) {
+				const cleanup = autoUpdate(referenceEl, floatingEl, update, {
+					// JSDOM does not support ResizeObserver
+					// https://floating-ui.com/docs/autoupdate#elementresize
+					elementResize: typeof ResizeObserver === 'function',
+				});
+				return cleanup;
+			},
+		}),
+	});
+
+	// Ensures the floating element remains anchored to its reference element when using `display: none` or `visibility: hidden`
+	// https://floating-ui.com/docs/react#anchoring
+	useEffect(() => {
```

Comment:
As we are (and always have been in Combobox) using CSS to conditionally display the popover, we need this effect. This was one of the reasons we had some issues in the Combobox component.

This is from their docs: https://floating-ui.com/docs/react#anchoring

<img width="757" alt="Screenshot 2023-10-11 at 10 50 10 am" src="https://github.com/steelthreads/agds-next/assets/6265154/398c3271-c165-4fc1-8dc9-e09f2bc79a76">